pepper-apis/src/main/java/org/broadinstitute/ddp/client/DsmClient.java

```diff
@@ -0,0 +1,148 @@
+package org.broadinstitute.ddp.client;
+
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.typesafe.config.Config;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.constants.RouteConstants.PathParam;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.dsm.ParticipantStatus;
+import org.broadinstitute.ddp.util.Auth0Util;
+import org.broadinstitute.ddp.util.RouteUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A client for DSM services, effectively just a wrapper around the HTTP calls. API paths use SparkJava's syntax for
+ * path parameters, the same syntax used in {@link RouteConstants.API}.
+ */
+public class DsmClient {
+
+    public static final String API_CANCERS = "/app/cancers";
+    public static final String API_DRUGS = "/app/drugs";
+    public static final String API_PARTICIPANT_STATUS = String.format(
+            "/info/participantstatus/%s/%s", PathParam.STUDY_GUID, PathParam.USER_GUID);
+    public static final int DEFAULT_TIMEOUT_SECS = 30;
+
+    private static final Logger LOG = LoggerFactory.getLogger(DsmClient.class);
+    private static final Gson gson = new Gson();
+
+    private final String baseUrl;
+    private final String jwtSecret;
+    private final String jwtSigner;
+    private final HttpClient client;
+
+    public DsmClient(Config cfg) {
+        this(cfg.getString(ConfigFile.DSM_BASE_URL),
+                cfg.getString(ConfigFile.DSM_JWT_SECRET),
+                cfg.getString(ConfigFile.DSM_JWT_SIGNER));
+    }
+
+    public DsmClient(String baseUrl, String jwtSecret, String jwtSigner) {
+        this.baseUrl = baseUrl;
+        this.jwtSecret = jwtSecret;
+        this.jwtSigner = jwtSigner;
+        this.client = HttpClient.newHttpClient();
```

Comment:
I noticed we use a few different libraries to do HTTP calls (e.g. `OkHttp`, apache's http library, etc). I think we should standardize on what library to use. For this implementation, I opted for the HTTP client in Java 11's standard library.